python/lsst/ctrl/bps/bps_core.py

```diff
@@ -44,11 +44,11 @@
 
 import lsst.log
 from lsst.daf.butler import Butler
+from lsst.daf.butler.core.utils import iterable
 from lsst.pipe.base.graph import QuantumGraph
-from lsst.pipe.base.graph import QuantumGraphTaskNodes
 from lsst.ctrl.bps.bps_config import BpsConfig
 from lsst.daf.butler.core.config import Loader
-from lsst.ctrl.bps.bps_draw import draw_networkx_dot, draw_qgraph_html
+from lsst.ctrl.bps.bps_draw import draw_networkx_dot
 
 # Graph property
 FILENODE = 0
```

Comment:
Outside of this ticket, but consider using a Enum object, its a bit safer and offers the ability to do `is` tests

Reply:
Won't change here, but the graph property names should be changing/growing soon and will look at this then.